keanu-project/src/main/java/io/improbable/keanu/templating/SequenceBuilder.java

```diff
@@ -23,6 +23,7 @@
     private static final String PROXY_LABEL_MARKER = "proxy_for";
     private VertexDictionary initialState;
     private Map<VertexLabel, VertexLabel> transitionMapping = Collections.emptyMap();
+    private String identifyingNamespace;
```

Comment:
nit: could you please rename this to `sequenceName`, as you did with the same field in `SequenceItem`